.github/workflows/release_npm_version.yml

```diff
@@ -44,25 +42,15 @@ jobs:
           echo "PACKAGE_VERSION=v${PACKAGE_VERSION}" >> $GITHUB_ENV
       
       - name: Create release
-        uses: actions/github-script@v6
-        with:
-          script: |
-            const name = 'Release Version: ${{ env.PACKAGE_VERSION }}';
-            const body = 'SDK Version ${{ env.PACKAGE_VERSION }}';
-            const createRelease = async () => {
-                await github.rest.repos.createRelease({
-                    owner: context.repo.owner,
-                    repo: context.repo.repo,
-                    tag_name: 'v${{ env.PACKAGE_VERSION }}',
-                    name: name,
-                    body: body,
-                    draft: false,
-                    prerelease: false,
-                    generate_release_notes: true
-                });
-                console.log(`Release created`);
-            }
-            createRelease();
+        run: |
+          gh release create ${{ env.PACKAGE_VERSION }} --generate-notes --latest --title "${{ env.PACKAGE_VERSION }}"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Create latest Tag
+        run: |
+            git tag latest
+            git push -f origin latest
```

Comment:
You will have to delete the existing latest/development tag otherwise it will error, look at https://github.com/Neuro-ID/neuroid-ios-sdk/blob/55610362b1a0db852624febea3be0bb760e6b790/.github/workflows/ios-sandbox-deployment-dev.yml#L63 for an example

Reply:
I didn't encounter errors while trying it my repo but will delete and create to be safe.